packages/docs/docs/cli/external-fhir-servers.md

```diff
@@ -19,34 +19,87 @@ To enable these scenarios, you will need a `clientId` or `clientSecret` to acces
 
 The examples below use the [CLI optional flags](/docs/cli#optional-flags).
 
+## Setting up your credentials
+
+Medplum CLI stores credentials to be used in a future period without needing it to be entered in every command. By using the `profile` command, this helps with the ability to work with multiple FHIR servers.
+
+### Example: Setting a Profile
+
+In this example, we will set up a profile using `medplum profile set <profileName>` with the flags below:
+
+- --auth-type
+- --client-id
+- --client-secret
+- --base-url
+- --token-url
+- --authorize-url
+- --fhir-url-path
+
+```bash
+medplum profile set <profileName> --base-url <base-url> --fhir-url-path <fhir-url-path> --token-url <token-url> --client-id <client-id> --client-secret <client-secret>
+```
+
+The profile will now be stored in a file directory in `~.medplum/<profileName>.json`
+
+Once you have a profile, you can connect with external FHIR servers with your profile using the `-p` flag. 
+
+
+Other profile commands include:
+#### `describe`
+
+To see the state of your credentials in on profile 
+
+Example:
```

Comment:
For each of these, let's do 2 blocks:

--------

Syntax:

```bash
medplum profile describe <profileName>
```

Example:

```bash
medplum profile describe "example"
```

--------

I know it's kinda verbose, but I think it's more explicit and technically correct.